sumeh/core/base/analyzer.py

```
@@ -0,0 +1,48 @@
+        >>>         # Compute metric
+        >>>         value = df[field].some_computation()
+        >>>         return MetricResult(
+        >>>             metric_type="my_metric",
+        >>>             field=field,
+        >>>             value=value,
+        >>>             total_rows=len(df),
+        >>>         )
+    """
+
+    @staticmethod
+    def analyze(data: Any, field: str, **kwargs) -> MetricResult:
```

Ruff (0.15.4)

[error] Undefined name `MetricResult`

(F821)